src/jit/emitxarch.cpp

```diff
@@ -1191,18 +1191,15 @@ inline ssize_t emitter::emitGetInsCIdisp(instrDesc* id)
  *  The following table is used by the instIsFP()/instUse/DefFlags() helpers.
```

Comment:
I wasn't able to find `instUseFlags` or `instDefFlags` (or anywhere that uses those `instInfo` fields).